print("This will show up in your output.log file")

raise Exception("This will show up in your error.log file")
